blockchain/common_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/btcsuite/btcd/blockchain/testhelper"
```

Comment:
Commit references `invalidateblock` and `reconsiderblock`, but only the latter was added in this PR.

Reply:
I think for cases like the issue with the pruning bug w/ the utxo set cache, we actually want `reconsiderblock`, as `invalidateblock` marks all the dependents as invalid, so it won't attempt to _reprocess_ anything after the target block.

Reply:
I'll add it in as another PR so that it doesn't add on to the needed reviews on this PR.

Was holding it off because the test for `reconsiderblock` utilizes `invalidateblock`